### PR TITLE
Add mojo parameter jsonConfigurationFile.

### PR DESCRIPTION
**Pull request by valinha**
Path to separate json configuration file. File content should be in a json format {optionKey:optionValue, optionKey1:optionValue1...}

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <groupId>io.github.kobylynskyi</groupId>
     <artifactId>graphql-codegen-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>1.2.4-SNAPSHOT</version>
+    <version>1.3.0</version>
```
Comment (kobylynskyi) [inline pom.xml]: Please update the version to 1.3.0-SNAPSHOT (maven-publish plugin requires that)

File: src/main/java/io/github/kobylynskyi/graphql/codegen/GraphqlCodegenMojo.java [hunk 2]
```diff
@@ -18,165 +20,177 @@
...
+    this.generateApis = generateApis;
+  }
+
+  public boolean isGenerateEqualsAndHashCode() {
+    return generateEqualsAndHashCode;
+  }
+
+  public void setGenerateEqualsAndHashCode(boolean generateEqualsAndHashCode) {
+    this.generateEqualsAndHashCode = generateEqualsAndHashCode;
+  }
```
Comment (kobylynskyi) [inline src/main/java/io/github/kobylynskyi/graphql/codegen/GraphqlCodegenMojo.java]: Please fix the formatting with 4 spaces (as it was before).

**Review by kobylynskyi — CHANGES_REQUESTED** [review]

**Comment by kobylynskyi** [comment]
@valinha are you planning to do the same changes for [graphql-java-codegen-gradle-plugin](https://github.com/kobylynskyi/graphql-java-codegen-gradle-plugin)?
If no, then I will do it.

**Comment by valinha** [comment]
@kobylynskyi I'm not very familiar with gradle plugins. If you can add your great, otherwise I check how to do it (it doesn't seem difficult)

**Comment by valinha** [comment]
> @valinha are you planning to do the same changes for [graphql-java-codegen-gradle-plugin](https://github.com/kobylynskyi/graphql-java-codegen-gradle-plugin)?
> If no, then I will do it.

I'm not very familiar with gradle plugins. If you can add your great, otherwise I check how to do it (it doesn't seem difficult)

File: src/main/java/io/github/kobylynskyi/graphql/codegen/GraphqlCodegenMojo.java (hunk 2 above)
Comment (valinha) [reply src/main/java/io/github/kobylynskyi/graphql/codegen/GraphqlCodegenMojo.java]: > Please fix the formatting with 4 spaces (as it was before).

Ok, sorry i had google style enabled. I will change it. I prefer 4 spaces too.

File: pom.xml (hunk 1 above)
Comment (valinha) [reply pom.xml]: Done

File: src/main/java/io/github/kobylynskyi/graphql/codegen/GraphqlCodegenMojo.java (hunk 2 above)
Comment (valinha) [reply src/main/java/io/github/kobylynskyi/graphql/codegen/GraphqlCodegenMojo.java]: Done

**Review by kobylynskyi — APPROVED** [review]

**Pull request merged**